docs/resources.md

```diff
@@ -422,7 +422,7 @@ spec:
     - name: url
       value: https://github.com/wizzbangcorp/wizzbang/pulls/1
   secrets:
-    - fieldName: githubToken
+    - fieldName: authToken
```

Comment:
hm this was a backwards incompatible change :thinking:  we made some other backwards incompatible changes to the pull request resource around the same time (e.g. the format of files like `status`), would have been great to roll these out in a backwards compatible way first

(this turned out to be the root of my sorrow in https://github.com/tektoncd/pipeline/issues/1777 :sob: )

+ @ImJasonH @wlynch